ledger/ledger-api-integration-tests/src/test/lib/scala/com/digitalasset/platform/apitesting/LedgerContext.scala

```diff
@@ -239,4 +242,47 @@ object LedgerContext {
     implicit def withResource(resource: Resource[SingleChannelContext]) =
       ResourceExtensions.MultiResource(resource)
   }
+
+
+  class MultiChannelContext(val mapping: Map[Party, LedgerContext],
+                            val defaultParty: Party,
+                            implicit val _esf: ExecutionSequencerFactory) extends LedgerContext {
```

Comment:
`implicits` should go to a second argument block

Reply:
Done